src/shared/ui/NotificationDropdown/NotificationDropdown.tsx

```
@@ -0,0 +1,251 @@
+};
+
+// 개별 알림 아이템 컴포넌트
+const NotificationItem: React.FC<{
+  notification: NotificationItem;
+  onClick: () => void;
+}> = ({ notification, onClick }) => {
+  const config = notificationConfig[notification.type];
+  const iconName = config.icon;
+
+  return (
+    <div
```

Biome (2.1.2)

[error] Shouldn't redeclare 'NotificationItem'. Consider to delete it or rename it.

'NotificationItem' is defined here:


(lint/suspicious/noRedeclare)